test/unit/mdc-select/foundation.test.js

```diff
@@ -164,15 +167,129 @@ test('#handleFocus calls adapter.activateBottomLine()', () => {
   td.verify(mockAdapter.activateBottomLine(), {times: 1});
 });
 
+test('#handleFocus does not call adapter.activateBottomLine() if isMenuOpen=true', () => {
+  const {foundation, mockAdapter} = setupTest();
+  td.when(mockAdapter.isMenuOpen()).thenReturn(true);
+  foundation.handleFocus();
+  td.verify(mockAdapter.activateBottomLine(), {times: 1});
```

Comment:
This test's verification was changed to contradict its description. What's the right answer?